aspnet/hosting/index.rst

```diff
@@ -0,0 +1,13 @@
+Hosting
+=======
+
+.. toctree::
+   :titlesonly:
+   
+   servicing
```

Comment:
The order is strange. It needs an introduction topic, then I'd suggest Installing Platform Handler, followed by Directory Structure, App Pools then Servicing.